domain/src/main/kotlin/com/xorker/draw/exception/XorkerException.kt

```diff
@@ -1,25 +1,26 @@
 package com.xorker.draw.exception
 
-sealed class XprException(val code: String, message: String, cause: Throwable? = null) : RuntimeException(message, cause)
+sealed class XorkerException(val code: String, message: String, cause: Throwable? = null) : RuntimeException(message, cause)
 
 //region Client
-sealed class ClientException(code: String, message: String, cause: Throwable? = null) : XprException(code, message, cause)
+sealed class ClientException(code: String, message: String, cause: Throwable? = null) : XorkerException(code, message, cause)
 
 data object UnAuthenticationException : ClientException("auth401", "인증 실패") { private fun readResolve(): Any = UnAuthenticationException }
 data object UnAuthorizedException : ClientException("auth403", "인가 실패") { private fun readResolve(): Any = UnAuthorizedException }
 
 data object InvalidRequestValueException : ClientException("c001", "Request 값 잘못됨") { private fun readResolve(): Any = InvalidRequestValueException }
 data object OAuthFailureException : ClientException("c002", "OAuth 인증 실패") { private fun readResolve(): Any = OAuthFailureException }
+data object NotFoundRoomException : ClientException("c003", "존재하지 않는 Room Id") { private fun readResolve(): Any = InvalidUserStatusException }
```

Comment:
P1
```suggestion
data object NotFoundRoomException : ClientException("c003", "존재하지 않는 Room Id") { private fun readResolve(): Any = NotFoundRoomException }
```
readResolve() 메서드에 오타가 있어서 수정 부탁드립니다.